testcases/kernel/syscalls/sync_file_range/sync_file_range02.c

```diff
@@ -21,13 +21,14 @@
 #include "lapi/sync_file_range.h"
 #include "check_sync_file_range.h"
 
-#define MNTPOINT		"mnt_point"
+#define MNTPOINT		"/data"
```

Comment:
This is not required. Please add below config parameters to tst_test test structure initialization in fdatasync03.c  file.
.mount_device = 1,
.all_filesystems = 1,
.mntpoint = MNTPOINT,

By enabling above options, test framework will mount the file system at the time of test initialization time.